libfreerdp/crypto/per.c

```diff
@@ -328,8 +329,8 @@ BOOL per_read_object_identifier(wStream* s, BYTE oid[6])
 		return FALSE;
 
 	Stream_Read_UINT8(s, t12); /* first two tuples */
-	a_oid[0] = (t12 >> 4);
```

Comment:
Where do you get the change from the shift by 4 (division by 16) to division by 40?

Reply:
Afaict this is described in https://msdn.microsoft.com/en-us/library/windows/desktop/bb540809 - the previous calculation with shifting seems to be wrong

Reply:
Ok, so that explains it, perfect.

Reply:
First tuple is encoded as 40m + n if m < 3 and n < 40, see:
https://msdn.microsoft.com/en-us/library/windows/desktop/bb540809
https://msdn.microsoft.com/en-us/library/cc240836.aspx

Sorry that I did not explain it fully in the commit message.